lib/identity_cache/query_api.rb

```diff
@@ -108,7 +108,7 @@ def prefetch_associations(associations, records)
 
       def raise_if_scoped
         if current_scope
-          IdentityCache.logger.error("#{name} has scope: #{current_scope.where_values_hash.inspect}")
+          IdentityCache.logger.error("#{name} has scope: #{current_scope.values.inspect}")
```

Comment:
What is in `values`?  Could calling inspect on it result in a slow query?  Should we instead just get `current_scope .to_sql` and `current_scope.values.keys` to avoid that possibility?